src/components/Card/Card.module.css

```diff
@@ -22,11 +22,11 @@
   background-color: var(--palette-common-white, #ffffff);
   border-radius: var(--shape-border-radius-lg, 8px);
   border-width: var(--shape-border-width-xs, 1px);
-  padding: var(--spacing-padding-sm, 8px) 0;
+  padding: var(--spacing-padding-sm, 8px) var(--spacing-padding-xl, 24px);
```

Comment:
could this cause issues on the already implemented cards on website? like double padding?